docs/docs/reference/configuration.md

```diff
@@ -76,6 +76,27 @@ Defines the bootstrap server of Kafka brokers to consume from. Multiple Kafka br
 * Example value: `localhost:9092`
 * Type: `required` only when `KAFKA_CONSUMER` or `KAFKA_SOURCE` is configured in `SOURCE_DETAILS`
 
+##### `SOURCE_KAFKA_CONSUMER_CONFIG_SECURITY_PROTOCOL`
+
+Defines the security protocol used to communicate with ACL enabled kafka. Valid values are: PLAINTEXT, SSL, SASL_PLAINTEXT, SASL_SSL.
+
+* Example value: `SASL_PLAINTEXT`
+* Type: `optional` required only for ACL enabled `KAFKA_CONSUMER` or `KAFKA_SOURCE` 
+
+##### `SOURCE_KAFKA_CONSUMER_CONFIG_SASL_MECHANISM`
```

Comment:
What is difference or relationship between `SOURCE_KAFKA_CONSUMER_CONFIG_SECURITY_PROTOCOL` and `SOURCE_KAFKA_CONSUMER_CONFIG_SECURITY_PROTOCOL` config ?

Is `SOURCE_KAFKA_CONSUMER_CONFIG_SASL_MECHANISM` only when `SOURCE_KAFKA_CONSUMER_CONFIG_SECURITY_PROTOCOL` is set to `SASL_SSL` or `SASL_PLAINTEXT` ?